src/bot/commands/random/dog.ts

```diff
@@ -0,0 +1,214 @@
+import { Command, Input, Listener } from '@api';
+import * as request from 'request';
+import { Response } from 'request';
+import { Message } from 'discord.js';
+import { RichEmbed } from 'discord.js';
+import { TextChannel } from 'discord.js';
+import { Emoji } from '@bot/libraries/emoji';
+import { Reactions } from '@bot/libraries/reactions';
+import { Framework } from '@core/framework';
+import { Timer } from '@bot/libraries/utilities/timer';
+import { ReactionListener } from '@bot/listeners/reactions';
+import { listeners } from 'cluster';
+import { GuildMember } from 'discord.js';
+import { Economy } from "@libraries/economy";
+import { debug } from 'util';
+import { Url } from 'url';
+const entities = require("html-entities").AllHtmlEntities;
+
```

Comment:
Just curious if there's any API's or any packages that allows the code to spit out a random image of a dog, what's the change that's been made to make the dog look more cuter? Does it show a user a better picture of a dog that's cuter or just any picture that seems appropriate for the bot?